docs/visualize/aggregations.asciidoc

```diff
@@ -0,0 +1,101 @@
+[[supported-aggregations]]
+=== Supported aggregations
+
+All of these visualizations support:
+
+**Metric Aggregations:**
+
+* {ref}/search-aggregations-metrics-avg-aggregation.html[Average]
```

Comment:
I added definitions for all of these except `Count`. I'm not sure what this means.

Reply:
Count is a special one because it really is just "count of documents," a number that represents how many documents are in each bucket.